### PR TITLE
Mentions stopping allergic reactions on medipens

### PR DESCRIPTION
**Pull request by lyricalpaws**
## About The Pull Request
Title

## Why It's Good For The Game
Because a lot of people didn't know that epinephrine stops production of histamine during allergic reactions.

## Changelog

:cl:
qol: Adds a new mention to the epinephrine pens about allergies
/:cl:

### CODE REVIEW

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 1]
```diff
@@ -99,8 +99,7 @@
 
 /obj/item/reagent_containers/hypospray/medipen
 	name = "epinephrine medipen"
-	desc = "A rapid and safe way to stabilize patients in critical condition for personnel without advanced medical knowledge. Contains a powerful preservative that can delay decomposition when applied to a dead body."
-	icon_state = "medipen"
+	desc = "A rapid and safe way to stabilize patients in critical condition for personnel without advanced medical knowledge. Contains a powerful preservative that can delay decomposition when applied to a dead body, and stop the production of histamine during an allergic reaction."	icon_state = "medipen"
```
Comment (GoldenAlpharex) [inline code/modules/reagents/reagent_containers/hypospray.dm:102]: I think you messed up on accident here.
```suggestion
	desc = "A rapid and safe way to stabilize patients in critical condition for personnel without advanced medical knowledge. Contains a powerful preservative that can delay decomposition when applied to a dead body, and stop the production of histamine during an allergic reaction."
	icon_state = "medipen"
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

**Comment by lyricalpaws** [comment]
Oops yeah that was a typo

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 2]
```diff
@@ -99,7 +99,7 @@
 
 /obj/item/reagent_containers/hypospray/medipen
 	name = "epinephrine medipen"
-	desc = "A rapid and safe way to stabilize patients in critical condition for personnel without advanced medical knowledge. Contains a powerful preservative that can delay decomposition when applied to a dead body."
+	desc = "A rapid and safe way to stabilize patients in critical condition for personnel without advanced medical knowledge. Contains a powerful preservative that can delay decomposition when applied to a dead body, and stop the production of histamine during an allergic reaction."
```
Comment (LemonInTheDark) [inline code/modules/reagents/reagent_containers/hypospray.dm:102]: I'm uh, kinda behind the times, but do we A: have allergies, and B: does the medipen actually fix them/how

Comment (lyricalpaws) [reply code/modules/reagents/reagent_containers/hypospray.dm:102]: Extreme medicine allergy is a quirk we have, and it stop the production of histamine for a time so you can go see a doctor ans not die.

**Review by LemonInTheDark — APPROVED** [review]

File: code/modules/reagents/reagent_containers/hypospray.dm (hunk 2 above)
Comment (LemonInTheDark) [reply code/modules/reagents/reagent_containers/hypospray.dm:102]: is it the preservative that stops the production of histamine? I guess there's not a great way to fix that

Comment (lyricalpaws) [reply code/modules/reagents/reagent_containers/hypospray.dm:102]: The formaldehyde stops organ decay, the ephedrine stops histamine production and stabilizes people in crit

**Pull request merged**